Formula/pyinvoke.rb

```diff
@@ -0,0 +1,38 @@
+class Pyinvoke < Formula
+  include Language::Python::Virtualenv
+
+  desc "Pythonic task management & command execution"
+  homepage "http://pyinvoke.org/"
+  url "https://github.com/pyinvoke/invoke/archive/0.13.0.tar.gz"
+  sha256 "4942aa901618f99afa70271d7b15fd018da0f174178fbca4048354c785e7736a"
+
+  head "https://github.com/pyinvoke/invoke.git"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    (testpath/"tasks.py").write <<-EOS.undent
+      from invoke import run, task
+
+      @task
+      def clean(ctx, extra=''):
+          patterns = ['foo']
+          if extra:
+              patterns.append(extra)
+          for pattern in patterns:
+              run("rm -rf {}".format(pattern))
+    EOS
+    mkdir_p testpath/"foo"/"bar"
+    mkdir_p testpath/"baz"
+    system bin/"invoke", "clean"
+    assert !File.exist?(testpath/"foo"), "\"pyinvoke clean\" should have deleted \"foo\""
```

Comment:
How does one `assert_predicate thing, *negated* operation`? I tried the simplistic `a_p thing, !:exist?` and got a horrible explosion and I didn't see any `:missing?` on `File`

I only found [one example](https://github.com/Homebrew/homebrew-core/blob/0379d0d/Formula/conan.rb#L141) across all the `homebrew-*` Taps I have locally, and [the documentation](http://ruby-doc.org/stdlib-2.0.0/libdoc/minitest/rdoc/MiniTest/Assertions.html#assert_predicate-method) wasn't helpful.